README.md

```diff
@@ -149,6 +149,33 @@ Examples:
         organization: "153963171798"
   ```
 
+### Defining inputs
+
+Data for cluster validation can be retrieved from multiple datasources,
+eg. GKE API, Kubernetes API, Cloud Monitoring API or local JSON file exported from GKE API.
+Check [Inputs user guide](./docs/user-guide.md#inputs)
```

Comment:
I think we should enable `gkeAPI` by default on "check and scalability check" commands.
The same, "metricsAPI" should be enabled by default on "scalability check" command.

Lets mention that in the docs as well (to avoid confusion)